src/helics/core/ActionMessage.hpp

```diff
@@ -157,11 +157,11 @@ class ActionMessage
 
     int serializedByteCount () const
     {
-        int size = 45;
+        int size = 45;  // 45 is the base size of the message
```

Comment:
Is there a struct or class that this number can be computed from, or is it arbitrary?

Reply:
It is a partial subset of the ActionMessage that gets serialized every time, so it isn't arbitrary.  It probably should be made clearer as to the origins.